diff --git a/node_modules/jest-chain/dist/chain.js b/node_modules/jest-chain/dist/chain.js
index 4a7b85b..a6edb0f 100644
--- a/node_modules/jest-chain/dist/chain.js
+++ b/node_modules/jest-chain/dist/chain.js
@@ -7,7 +7,7 @@ exports.default = void 0;
 
 class JestAssertionError extends Error {
   constructor(result, callsite) {
-    super(result.message());
+    super(typeof result.message === "function" ? result.message() : result.message);
     this.matcherResult = result;
 
     if (Error.captureStackTrace) {
